### PR TITLE
Restaurant description

### PR DESCRIPTION
**Pull request by lcegit**
## PT Link: https://www.pivotaltracker.com/story/show/154132089

### CODE REVIEW

**Review by tochman — CHANGES_REQUESTED** [review]
I see unfomatted code and coverage results checked in again

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
```
Comment (oliverochman) [inline db/schema.rb]: maybe removing the stuff that is commented out

File: features/step_definitions/restaurant_list_steps.rb [hunk 2]
```diff
@@ -3,3 +3,19 @@
     FactoryBot.create(:restaurant, restaurant)
   end
 end
+
+Then("show me the page") do
```
Comment (oliverochman) [inline features/step_definitions/restaurant_list_steps.rb:7]: This step definition should be removed

Comment (lcegit) [reply features/step_definitions/restaurant_list_steps.rb:7]: This has been removed in feature file but would prefer to keep it here for future step defs. Thanks!

**Comment by lcegit** [comment]
Coverage files have been omitted. @tochman

**Review by diraulo — COMMENTED** [review]
Why does this PR have the same code as in #5 🤔

**Comment by lcegit** [comment]
We're aware of the overlap, this PR is dependent on #5. Please advise if we need to cherry pick commits. @diraulo

**Comment by diraulo** [comment]
if #5 is ready then let's get that in first before the cherry-pick

**Comment by lcegit** [comment]
@diraulo Great. This PR should be mergeable now.

**Comment by diraulo** [comment]
This PR also got merged automatically since it had the same commits as the previous one...

**Pull request merged**